src/app/account/identify/identify.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component } from '@angular/core';
+import { Validators, FormControl } from '@angular/forms';
+
+import { LOGIN_ERROR_MESSAGES } from '@constants/app-constants';
+
+import { AuthService } from 'app/shared/services/auth.service';
+
+@Component({
+  selector: 'app-identify',
+  templateUrl: './identify.component.html',
+  styleUrls: ['./identify.component.scss']
+})
+export class IdentifyComponent {
+
+  public emailFormControl = new FormControl('', [
+    Validators.required,
+    Validators.email
+  ]);
+  public errorMessage: string;
+  public showError = false;
+
+  constructor(private authService: AuthService) {}
+
+  onSubmit(sendOTP: boolean) {
```

Comment:
```suggestion
  onSubmit(createOTP: boolean) {
```

Reply:
This boolean is used to decide that we should send an email or not. In any case, OTP is created.